components/tools/OmeroJava/test/integration/LightAdminRolesTest.java

```diff
@@ -97,13 +99,15 @@
 import org.testng.annotations.Test;
 
 import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
 import com.google.common.collect.ImmutableSet;
 
 /**
  * Tests the concrete workflows of the light admins
  * @author p.walczysko@dundee.ac.uk
  * @since 5.4.0
  */
+
```

Comment:
It's unusual to have a blank line in between the Javadoc and what it's Javadoc for.